test/php/library/Icinga/Util/LessParserTest.php

```diff
@@ -60,6 +60,38 @@ public function testSimpleVariables()
         );
     }
 
+    public function testSimpleNestedVariables()
+    {
+        $this->assertEquals(
+            <<<CSS
+.black {
+  color: var(--my-color, var(--black, #000000));
+}
+.notBlack {
+  color: var(--my-black-color, var(--my-color, var(--black, #000000)));
+}
+
+CSS
+            ,
+            $this->compileLess(<<<LESS
+@black: black;
+@my-color: @black;
+@my-black-color: @my-color;
+// This triggers invalid argument exception without our custom call class
+@fade-color: fade(@my-color, 50%);
```

Comment:
Please create a separate test case for this. And don't just test for the non-error, but check the result as well.